src/style/images-and-videos.module.css

```
@@ -82,4 +86,60 @@
 
 .thumbnail-container::-webkit-scrollbar {
     width: 8px;
-}
\ No newline at end of file
+}
+
+/* smaller sizes on computer screen */
+@media (min-width: 1280px) and (max-width: 1920px) {
+    .container :global(.ant-card-body) {
+        display: flex;
+        margin: 24px 0 40px 0;
+        padding: 0 24px;
+        gap: 0;
+        justify-content: space-between;
+    }
+
+    .primary-image-only {
+        max-width: 100%;
+    }
+
+    .primary-image-with-thumbnail {
+        max-width: 100%;
+        flex: 1 1 auto;
+    }
+}
+
+/* tablet sizing */
+@media (max-width: 1280px) {
+    .container {
+        max-width: 960px;
+        min-width: 0;
+        height: auto;
+    }
+
+    .container .primary-image-container :global(.gatsby-image-wrapper) {
+        height: auto;
+    }
+
+    .container :global(.ant-card-body) {
+        margin: 20px 0px;
+        gap: 20px;
+        justify-content: space-between;
+    }
+
+    .caption {
+        max-height: auto;
```

```suggestion
        max-height: min-content;
```
this is not working on chrome